src/libopensc/card-idprime.c

```diff
@@ -452,6 +456,7 @@ static int idprime_process_index(sc_card_t *card, idprime_private_data_t *priv,
 				new_object.key_reference = 0x41 + cert_id;
 				break;
 			case SC_CARD_TYPE_IDPRIME_930:
+			case SC_CARD_TYPE_IDPRIME_930_2:
```

Comment:
Dunno if this is correct case? If it works then it is?

Reply:
Correct.